src/pages/projects/index.page.tsx

```diff
@@ -86,6 +94,17 @@ function Projects() {
     return _pv.length ? _pv : [PV_V1, PV_V2]
   }, [includeV1, includeV2])
 
+  function updateRoute(
+    _searchTags: ProjectTag[],
+    _searchText: string | undefined,
+  ) {
+    router.push(
+      `/projects?tab=all${_searchText ? `&search=${_searchText}` : ''}${
+        _searchTags.length ? `&tags=${_searchTags.join(',')}` : ''
+      }`,
```

Comment:
**suggestion** might be preferable to use `qs` for this

Reply:
done, great call